modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -94,13 +108,40 @@
      * Bind dependant fields
      */
     FormWidget.prototype.bindDependants = function() {
+        var self = this,
+            fieldMap = this._getDependants()
 
+        /*
+         * When a master is updated, refresh its slaves
```

Comment:
@msimkunas Just to make it more clear what is happening (and ward off the PC crowd)
```suggestion
         * When a field is updated, refresh its dependents
```

Reply:
This should also be updated in the existing code as well: https://github.com/octobercms/october/blob/develop/modules/backend/widgets/form/assets/js/october.form.js#L106-L131